velox/functions/lib/tests/SliceTest.cpp

```diff
@@ -126,6 +133,32 @@ TEST_F(SliceTest, prestoTestCases) {
   }
 }
 
+TEST_F(SliceTest, sparkTestCases) {
+  auto arrayVector = makeArrayVector<int64_t>({{1, 2, 3, 4, 5, 6}});
+  auto expectedArrayVector = makeArrayVector<int64_t>({{1, 2, 3, 4}});
+  // Spark Slice() needs to parse integer literals as INTEGER, not BIGINT.
```

Comment:
-> "Parses integer literals as INTEGER, not BIGINT".

You can keep this comment for `options_.parseIntegerAsBigint = false;`.